packages/lib/src/asset-manager/asset-manager.wc.svelte

```diff
@@ -117,6 +129,14 @@
   }
 
   function confirmSelection() {
+    if (Object.keys(selectedItems).length < minSelected){
+      console.log("Please select a file first.");
+      return;
+    }
+    if (Object.keys(selectedItems).length >= maxSelected) {
+      console.log(`Too many files have been selected. The maximum allowed is ${maxSelected}.`);
+      return;
+    }
```

Comment:
Since neither `minSelected` nor `maxSelected` is required you need to also first check if they are provided and only do this validation if they are.